pkg/ui/installsupport.go

```diff
@@ -32,8 +32,8 @@ func InstallSupport(mux MuxInterface) {
 	prefix := "/static/"
 	mux.Handle(prefix, http.StripPrefix(prefix, fileServer))
 
-	// Expose files in third_party/swagger-ui/ on <host>/swagger-ui/
+	// Expose files in third_party/swagger-ui/ on <host>/
 	fileServer = http.FileServer(&assetfs.AssetFS{Asset: Asset, AssetDir: AssetDir, Prefix: "third_party/swagger-ui"})
-	prefix = "/swagger-ui/"
+	prefix = "/"
```

Comment:
We shouldn't expose this on `/`. For one thing, the swagger support is optional. Speaking of which, this should be optional, similar to: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/master/master.go#L416